pandas/compat/numpy/function.py

```diff
@@ -225,16 +226,32 @@ def validate_cum_func_with_skipna(skipna, args, kwargs, name):
 STAT_FUNC_DEFAULTS = OrderedDict()
 STAT_FUNC_DEFAULTS['dtype'] = None
 STAT_FUNC_DEFAULTS['out'] = None
+
+PROD_DEFAULTS = SUM_DEFAULTS = STAT_FUNC_DEFAULTS.copy()
+SUM_DEFAULTS['keepdims'] = False
+SUM_DEFAULTS['initial'] = None
+
+MEDIAN_DEFAULTS = STAT_FUNC_DEFAULTS.copy()
+MEDIAN_DEFAULTS['overwrite_input'] = False
+MEDIAN_DEFAULTS['keepdims'] = False
+
+STAT_FUNC_DEFAULTS['keepdims'] = False
+
 validate_stat_func = CompatValidator(STAT_FUNC_DEFAULTS,
                                      method='kwargs')
-validate_sum = CompatValidator(STAT_FUNC_DEFAULTS, fname='sort',
+validate_sum = CompatValidator(SUM_DEFAULTS, fname='sum',
```

Comment:
I think the incorrect `fname` was a copy-paste error.

Reply:
Good catch 🙂